### PR TITLE
Web Update 1 - Style

### PR DESCRIPTION
**Pull request by trisyoungs**
This PR updates the style of the website ahead of the large content dump in a subsequent PR.

It reorganises the main page, splitting downloads onto a separate page, adds a multitude of Hugo shortcodes, and updates the styling of several elements.

### CODE REVIEW

File: web/config.toml [hunk 1]
```diff
@@ -46,6 +46,7 @@ latexDashes = true
 [params]
 copyright = "Dr Tristan Youngs, Simona. Built using Hugo/Docsy."
```
Comment (DanNixon) [inline web/config.toml:47]: Should this be changed to match the copyright note in the SPDX comments?

File: web/assets/scss/_content.scss [hunk 2]
```diff
@@ -267,3 +267,5 @@ table {
 a {
 	color: #454a8a;
 }
+=======
```
Comment (DanNixon) [inline web/assets/scss/_content.scss]: ?

File: web/content/packages.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+{{< blocks/section color="secondary" >}}
+<div class="mx-auto">
+<h2>Development Version {{< continuousversion >}}</h2>
+</div>
+{{< /blocks/section >}}
+
+{{< blocks/section color="secondary" >}}
+
+<div class="mx-auto">
+<p>These packages are built dynamically from the <a href="https://github.com/projectdissolve/dissolve">development branch</a> and as such contain the most up-to-date functionality, but may also contain bugs and other undocumented features. If you use these versions and discover a bug, please consider supporting the project and <a href="https://github.com/projectdissolve/dissolve/issues/new/choose">submitting an issue</a> to highlight it. Please include the unique hash value of the version you were using in your report.</p>
```
Comment (DanNixon) [inline web/content/packages.md:68]:
```suggestion
<p>These packages are built dynamically from the <a href="https://github.com/projectdissolve/dissolve">development branch</a> and as such contain the most up-to-date functionality, but may also contain bugs and other undocumented features. If you use these versions and discover a bug, please <a href="https://github.com/projectdissolve/dissolve/issues/new/choose">submit an issue</a> to highlight it. Please include the unique hash value of the version you were using in your report.</p>
```

"supporting the project" reads like wanting financial support to me.

**Review by DanNixon — APPROVED** [review]

File: web/config.toml (hunk 1 above)
Comment (trisyoungs) [reply web/config.toml:47]: Yep, that would be more consistent.

File: web/assets/scss/_content.scss (hunk 2 above)
Comment (trisyoungs) [reply web/assets/scss/_content.scss]: Odd. That isn't in my file, and GH has marked it as `Outdated` in the PR...

Comment (DanNixon) [reply web/assets/scss/_content.scss]: Yes, I was reviewing it per commit and this was removed after.

**Pull request merged**